src/sos/parser.py

```diff
@@ -1474,11 +1473,10 @@ def workflow(self,
                 wf_name = "default"
             else:
                 raise ValueError(
-                    "Name of workflow should be specified because "
-                    "the script defines more than one pipelines without a default one. "
-                    "Available pipelines are: {}.".format(", ".join(
-                        self.workflows)))
-        elif wf_name not in self.workflows and wf_name != "default":
+                    f"Name of workflow should be specified because "
+                    f"the script defines more than one pipelines without a default one. "
+                    f"Available pipelines are: {', '.join(self.workflows)}.")
+        if wf_name not in self.workflows and wf_name != "default":
```

Comment:
You again changed the logic of the code.

Reply:
I just checked my latest commit on https://github.com/jenningsje/Vat_sos/blob/consider-using-f-string/src/sos/parser.py. The two f'strings in lines 1476 and 1477 above have been replaced with normal strings.